browser/field-editor.js

```diff
@@ -170,16 +178,16 @@ function updateAddRemove() {
 
   // Remove existing adders and removers
   $("a[class|=add]").remove()
-  $("a[class|=remove]").remove()
+  //$("a[class|=remove]").remove()
```

Comment:
Why aren't you removing the existing removers? Won't this lead to lots of removers getting added?

Reply:
I conditioned the other code to prevent them getting added.  Granted, this was a bit wonky since I really don't understand browser code well.  It is also likely why I reload pages all the time.